SeaBattle/Ship.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class BaseShip
+    {
+        public BaseShip (string Name, List<PointStruct> points) { }
+        private List<Deck> Decks { get; set; }
+        int Durability;
+        int MoveSpeed;
+        int MoveDirectionX;
+        int MoveDirectionY;
+
+    }
+
+    public class Deck
```

Comment:
this file contains 6 classes and 2 interfaces, why?

Reply:
Because it Base Abstract class, or I need to split them ? Like Battleship.cs/ServiceShip.cs/HybridShip.cs etc..?

Reply:
of course, you have to. new type in the new file